### DISCUSSION TITLE
Ability to add labels to project members

### DISCUSSION THREAD

**Discussion by vasymus** [community/community#36315] [Projects and Issues]
I think it would be great to add labels of some kind to project members.

For example, `frontend developer`, `backend developer`, `product owner`, `tester` etc

**Comment by DSDmark** [comment]
### You can't do that, but instant of this you can try **Badges**
like that:-

![](https://img.shields.io/badge/develope-frontend-red)
![](https://img.shields.io/badge/develope-backend-green)
![](https://img.shields.io/badge/develope-fullstack-yellow)

![Testing-Library](https://img.shields.io/badge/-TestingLibrary-%23E33332?style=for-the-badge&logo=testing-library&logoColor=white)

#### Here is code, try copy and pest it in **.md** file.
```![](https://img.shields.io/badge/develope-frontend%20develope-red)```

#### For more information, check this out :-

[**shields.io**](https://shields.io)
[**ileriayo**](https://ileriayo.github.io/markdown-badges/)

**Comment by ossanna16** [comment]
Hey there @vasymus and thank you for the suggestion! I'm glad you found our community 🙂
